aten/src/ATen/native/cuda/Normalization.cuh

```diff
@@ -787,6 +810,8 @@ Tensor batch_norm_backward_elemt_cuda_template(const Tensor& grad_out_, const Te
                                                const Tensor& mean_, const Tensor& invstd_,
                                                const Tensor& weight_, const Tensor& mean_dy_, const Tensor& mean_dy_xmu_) {
 
+  if (input_.size(0) == 0) return at::empty_like(input_); // empty batch
```

Comment:
This will break double backward, because returned gradient will have no history (yes, I understand there may or may not be a single person on earth who is using SyncBatchNorm with empty batches and double backward)

Reply:
discussed offline.
We don't support double backward in SyncBN. If we want to add support for double backward, we should start another PR. I'm marking this as resolved.